specs/engage-digital_openapi3.yaml

```
@@ -810,6 +810,22 @@ paths:
           required: false
           schema:
             type: integer
+        - description: this field defines the time before a task is auto completed (in seconds).

```

There is an inconsistency in this document about starting descriptions with upper case or not, let's do uppercase like most of them imo.

```suggestion
        - description: Time before a task is auto completed (in seconds).
```

---

Done in a64b96752629b871a65d9886b428374c3a6777e4.